generators/dotnet-templates/Microsoft.BotFramework.CSharp.CoreBot/content/.template.config/template.json

```diff
@@ -45,16 +45,12 @@
         "datatype": "choice",
         "choices": [
           {
-            "choice": "netcoreapp2.2",
-            "description": "Target netcoreapp2.2"
-          },
-          {
-            "choice": "netcoreapp2.1",
-            "description": "Target netcoreapp2.1"
+            "choice": "netcoreapp3.1",
+            "description": "Target netcoreapp3.1"
           }
         ],
         "replaces": "__NETCOREAPP_VERSION__",
-        "defaultValue": "netcoreapp2.2"
+        "defaultValue": "netcoreapp3.1"
```

Comment:
Based on my feedback above, we can decide if 3.1 is the default or if we're going to ease into this.  Right now the command line arguments are incongruent with their intended purpose

Reply:
Agreed - this is going to depend on the outcome of my comment above.

Reply:
As per above - leave it with me.